src/man/sssd-kcm.8.xml

```diff
@@ -249,6 +280,19 @@ systemctl restart sssd-kcm.service
                     </para>
                 </listitem>
             </varlistentry>
+            <varlistentry>
+                <term>krb5_renew_interval (integer)</term>
+                <listitem>
+                    <para>
+                        The time in seconds between TGT renewal attempts for
+                        all KCM-stored renewable tickets. Renewals only trigger
+                        when about half of their ticket lifetime is exceeded.
+                    </para>
+                    <para>
+                        Default: 0 (Automatic renewal is disabled)
+                    </para>
+                </listitem>
+            </varlistentry>
```

Comment:
Remaining options are undocumented intentionally?

Reply:
I tried to keep the man page more concise. Above I provide the configurable krb5_* options list and state that sssd-krb5 contains the descriptions of each.

If you prefer I am happy to document each option in this man page.